snippets/remote_execution_ssh_keys.erb

```diff
@@ -32,4 +32,7 @@ EOF
 chmod 700 <%= ssh_path %>
 chmod 600 <%= ssh_path %>/authorized_keys
 chown -R <%= "#{ssh_user}:" %> <%= ssh_path %>
+
+# Restore SELinux context if a linux system with restorecon:
+command -v restorecon && restorecon -RvF <%= ssh_path %>
```

Comment:
I guess you're getting the exit code 1 from `command` since restorecon doesn't exist? I can refactor it so it's an if statement instead to avoid that I guess, but does this affect the code of any of the actual finish scripts?